plugins/statistics.koplugin/main.lua

```diff
@@ -1016,11 +1016,11 @@ function ReaderStatistics:getCurrentStat(id_book)
         { _("Total time"), util.secondsToClock(total_time_book, false) },
         { _("Total highlights"), tonumber(highlights) },
         { _("Total notes"), tonumber(notes) },
+        { _("Total read pages"), tonumber(total_read_pages) },
```

Comment:
Total pages read ought to sound better.
```suggestion
        { _("Total pages read"), tonumber(total_read_pages) },
```

Reply:
Which made me realize that `Current pages` sounds iffy, too. `Current pages read`?

Reply:
I'd also possibly swap "period" for "session", and repeat it where appropriate (i.e., "Pages read this session", "Current session duration", ...).

Reply:
"Time spent reading today", "Pages read today"